qa/L0_backend_python/lifecycle/lifecycle_test.py

```diff
@@ -98,6 +98,39 @@ def test_infer_pymodel_error(self):
                     False,
                     "Wrong exception raised or did not raise an exception")
 
+    def test_incorrect_execute_return(self):
+        model_name = 'execute_return_error'
+        shape = [1, 1]
+        with httpclient.InferenceServerClient("localhost:8000") as client:
+            input_data = (5 * np.random.randn(*shape)).astype(np.float32)
+            inputs = [
+                httpclient.InferInput("INPUT", input_data.shape,
+                                      np_to_triton_dtype(input_data.dtype))
+            ]
+            inputs[0].set_data_from_numpy(input_data)
+
+            # The first request to this model will return None.
+            with self.assertRaises(InferenceServerException) as e:
+                client.infer(model_name, inputs)
+
+            self.assertTrue(
```

Comment:
Look good! Using assertIn or assertEqual could result in a clearer failure message.